src/main/java/com/coveros/selenified/services/HTTP.java

```diff
@@ -20,6 +20,8 @@
 
 package com.coveros.selenified.services;
 
+import com.coveros.selenified.exceptions.InvalidProxyException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.coveros.selenified.exceptions.InvalidProxyException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)